<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="barber_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/barber_sleep.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="b1_text">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Barber 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="b2_text">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Barber 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="barber_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/barber_sleep.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="barber_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pic/barber_sleep.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="b3_text">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Barber 3</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
